microsite/sidebars.json

```diff
@@ -408,6 +408,11 @@
           "frontend-system/architecture/references"
         ]
       },
+      {
+        "type": "category",
+        "label": "Building Apps",
```

Comment:
```suggestion
        "label": "Migrating Apps",
```

Reply:
Nah this is in the apps section, just that the other content wasn't there yet